src/components/explorer/IndividualBiosamples.js

```diff
@@ -51,7 +51,7 @@ const IndividualBiosamples = ({individual}) =>
            pagination={{pageSize: 25}}
            columns={BIOSAMPLE_COLUMNS}
            rowKey="id"
-           dataSource={(individual || {}).biosamples || []} />;
+           dataSource={(individual || {}).phenopackets.map(p => p.biosamples).flat() || []} />;
```

Comment:
this could be shortened to `.flatMap(...`

Reply:
also, this can trip an error, since the `|| []` 
thing will not kick in if `.map` throws an error
however, i dont think its needed, since these fields should be always present, so the `|| []` at the end could probably just be removed